src/drivers/gitlab.go

```diff
@@ -8,12 +8,14 @@ import (
 	"github.com/git-town/git-town/src/drivers/helpers"
 )
 
+// gitlabConfig defines the configuration data needed by the GitLab driver.
 type gitlabConfig interface {
```

Comment:
Should this be replaced by the new shared `config`?

Reply:
Yes, that was an omission.